trivia-game/client/src/components/questioncard.js

```diff
@@ -0,0 +1,37 @@
+import {decode} from 'html-entities';
+import { useState} from "react";
+import Scoreboard from "./scoreboard";
+
+const QuestionCard = (props) => {
+    const [answered, setAnswered] = useState(false)
+    const [score,setScore] = useState(0);
+    const [userAnswer, setUserAnswer] = useState();
+    
+    const handleAnswerOptionClick = (userAnswerParam) => {
+      setUserAnswer(userAnswerParam);
+    if (userAnswerParam === props.question.correct_answer) {
+      props.setScore(props.score + 1);
+      setAnswered(true);
+      console.log(props.score)
```

Comment:
Just a reminder to try to remove all `console.log` statements before you put your PRs up for review. In a work setting, you do not want those going into production.